spring-ai-ragas-spring-boot-starter:0.3.1 POM не тянет транзитивные зависимости

## Проблема

`spring-ai-ragas-spring-boot-starter:0.3.1` — пустой POM: только `<parent>` без блока `<dependencies>`. Добавление одного стартера, как описано в README, не подтягивает `-metrics`, `-multi-model`, `-spring-boot` — и проект падает на этапе компиляции.

## Воспроизведение

1. Создать новый Spring Boot проект на Java 17+, Spring Boot 3.5.x.
2. Добавить в `pom.xml` ровно те зависимости, что указаны в [README.ru.md → Установка → Maven](https://github.com/ai-qa-solutions/spring-ai-ragas/blob/main/README.ru.md#maven):

```xml
<dependency>
    <groupId>io.github.ai-qa-solutions</groupId>
    <artifactId>spring-ai-ragas-spring-boot-starter</artifactId>
    <version>0.3.1</version>
    <scope>test</scope>
</dependency>
```

3. В тесте импортировать любую метрику, например `AspectCriticMetric`.
4. Запустить `mvn test-compile`.

## Ожидаемое поведение

Стартер должен тянуть все модули, необходимые для работы метрик: `spring-ai-ragas-spring-boot`, `spring-ai-ragas-metrics`, `spring-ai-ragas-multi-model`. Это стандартное поведение Spring Boot стартера.

## Фактическое поведение

```
[ERROR] package ai.qa.solutions.metrics.general does not exist
[ERROR] package ai.qa.solutions.metrics.agent does not exist
[ERROR] package ai.qa.solutions.metrics.retrieval does not exist
[ERROR] package ai.qa.solutions.sample does not exist
[ERROR] package ai.qa.solutions.sample.message does not exist
[ERROR] cannot find symbol: class AspectCriticMetric
[ERROR] cannot find symbol: class RubricsScoreMetric
...
```

## Содержимое опубликованного POM

```xml
<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
    <modelVersion>4.0.0</modelVersion>
    <parent>
        <groupId>io.github.ai-qa-solutions</groupId>
        <artifactId>spring-ai-ragas</artifactId>
        <version>0.3.1</version>
    </parent>

    <artifactId>spring-ai-ragas-spring-boot-starter</artifactId>

    <name>Spring AI RAGAS Spring Boot Starter</name>
    <description>Spring Boot starter for RAGAS metrics evaluation</description>
    <url>https://github.com/ai-qa-solutions/spring-ai-ragas</url>

</project>
```

Блок `<dependencies>` отсутствует полностью. Та же проблема в `1.0.0`.

## Обходной путь

Добавлять явно все необходимые модули:

```xml
<dependency>
    <groupId>io.github.ai-qa-solutions</groupId>
    <artifactId>spring-ai-ragas-spring-boot-starter</artifactId>
    <version>0.3.1</version>
    <scope>test</scope>
</dependency>
<dependency>
    <groupId>io.github.ai-qa-solutions</groupId>
    <artifactId>spring-ai-ragas-spring-boot</artifactId>
    <version>0.3.1</version>
    <scope>test</scope>
</dependency>
<dependency>
    <groupId>io.github.ai-qa-solutions</groupId>
    <artifactId>spring-ai-ragas-metrics</artifactId>
    <version>0.3.1</version>
    <scope>test</scope>
</dependency>
<dependency>
    <groupId>io.github.ai-qa-solutions</groupId>
    <artifactId>spring-ai-ragas-multi-model</artifactId>
    <version>0.3.1</version>
    <scope>test</scope>
</dependency>
```

При таком составе компиляция и все тесты проходят корректно (проверено на демо-проекте с GigaChat-2-Max).

## Предлагаемое исправление

Добавить в `spring-ai-ragas-spring-boot-starter/pom.xml` блок зависимостей по образцу Spring Boot starters — транзитивно подтягивать `-spring-boot` (который уже тянет `-metrics` и `-multi-model`):

```xml
<dependencies>
    <dependency>
        <groupId>io.github.ai-qa-solutions</groupId>
        <artifactId>spring-ai-ragas-spring-boot</artifactId>
    </dependency>
</dependencies>
```

И выпустить патч-версию `0.3.2`.

## Окружение

- `spring-ai-ragas`: `0.3.1` (Maven Central)
- Spring Boot: `3.5.11`
- Spring AI: `1.1.2`
- Maven: `3.9.11`
- Java: `21.0.10` (Temurin)

--- a/README.ru.md
+++ b/README.ru.md
@@ -316,7 +316,7 @@ public class GigaChatService {
 
 #### Поддерживаемые внешние стартеры
 
-| Стартер   | Артефакт                                     | ChatModel                                                            | EmbeddingModel              |
+|  Стартер  |                   Артефакт                   |                              ChatModel                               |       EmbeddingModel        |
 |-----------|----------------------------------------------|----------------------------------------------------------------------|-----------------------------|
 | GigaChat  | `chat.giga:spring-ai-starter-model-gigachat` | GigaChat, GigaChat-Pro, GigaChat-Max, GigaChat-2-Pro, GigaChat-2-Max | Embeddings, EmbeddingsGigaR |
 | Anthropic | `spring-ai-starter-model-anthropic`          | Claude                                                               | -                           |
@@ -357,7 +357,7 @@ LLM-as-judge даёт нестабильные оценки, если крите
 
 Метрики возвращают значения в разных диапазонах — пороги надо подбирать с учётом этого:
 
-| Метрика                                                                  | Диапазон                                 | Рекомендуемый порог                        |
+|                                 Метрика                                  |                 Диапазон                 |            Рекомендуемый порог             |
 |--------------------------------------------------------------------------|------------------------------------------|--------------------------------------------|
 | `AspectCriticMetric`, `ToolCallAccuracyMetric`                           | бинарный 0 / 1                           | `isEqualTo(1.0)`                           |
 | `SimpleCriteriaScoreMetric`                                              | [0..1] (нормализуется из raw)            | `>= 0.75` (raw 4/5) или `== 1.0` (raw 5/5) |

--- a/pom.xml
+++ b/pom.xml
@@ -97,6 +97,11 @@
             </dependency>
             <dependency>
                 <groupId>io.github.ai-qa-solutions</groupId>
+                <artifactId>spring-ai-ragas-spring-boot</artifactId>
+                <version>${project.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.github.ai-qa-solutions</groupId>
                 <artifactId>spring-ai-ragas-allure</artifactId>
                 <version>${project.version}</version>
             </dependency>

--- a/spring-ai-ragas-spring-boot-starter/pom.xml
+++ b/spring-ai-ragas-spring-boot-starter/pom.xml
@@ -15,4 +15,18 @@
     <description>Spring Boot starter for RAGAS metrics evaluation</description>
     <url>https://github.com/ai-qa-solutions/spring-ai-ragas</url>
 
+    <dependencies>
+        <!--
+             Pulls in the autoconfiguration module, which transitively brings
+             spring-ai-ragas-metrics, spring-ai-ragas-multi-model, bucket4j-core
+             and Spring Boot autoconfigure support. Adding this starter alone
+             must be enough to use any RAGAS metric in a Spring Boot project.
+             Version is managed by the parent pom's dependencyManagement.
+        -->
+        <dependency>
+            <groupId>io.github.ai-qa-solutions</groupId>
+            <artifactId>spring-ai-ragas-spring-boot</artifactId>
+        </dependency>
+    </dependencies>
+
 </project>

